ci/filter_example_mirror.sh

```diff
@@ -14,13 +15,17 @@ then
   git -C "$CHECKOUT_DIR" fetch
   git -C "$CHECKOUT_DIR" checkout -B master origin/master
 
+  echo "Updating Submodule..."
   # Update submodule to latest Envoy SHA
   ENVOY_SHA=$(git rev-parse HEAD)
   git -C "$CHECKOUT_DIR" submodule update --init
   git -C "$CHECKOUT_DIR/envoy" checkout "$ENVOY_SHA"
-  git -C "$CHECKOUT_DIR" commit -a -m "Update Envoy submodule to $ENVOY_SHA"
 
-  echo "Pushing..."
+  echo "Updating Workspace file."
+  sed -e "s|{ENVOY_SRCDIR}|${ENVOY_SRCDIR}|" "${ENVOY_SRCDIR}"/ci/WORKSPACE.filter.example > "${CHECKOUT_DIR}"/WORKSPACE
```

Comment:
`sed -e "s|{ENVOY_SRCDIR}|envoy|"`, that is fixed in filter-example repo as it uses submodules.

Reply:
Ah yes. Good catch.